api/src/models/master/plan.js

```diff
@@ -0,0 +1,30 @@
+'use strict';
+
+module.exports = require('../../utils/model-schema-generator')(
+  'Plan',
+  {
+    _schemaName: { type: String, default: 'Plan', index: true },
+    _epicProjectId: { type: 'ObjectId', default: null, index: true },
+    _sourceRefId: { type: 'ObjectId', default: null, index: true },
+    _epicMilestoneId: { type: 'ObjectId', default: null, index: true },
+
+    read: [{ type: String, trim: true, default: 'sysadmin' }],
+    write: [{ type: String, trim: true, default: 'sysadmin' }],
+
+    recordName: { type: String, default: '' },
+    recordType: { type: String, default: '' },
```

Comment:
Do we still have recordType?  I know there was some confusion before around 'type' since Lena/Kyle assumed it was a field, and we assumed it was the schema.  But maybe this is unrelated to that?

Reply:
Unrelated - it's just a field on these particular documents.